web-console/src/views/query-view/query-output/query-output.scss

```diff
@@ -22,6 +22,7 @@
     top: 0;
     bottom: 0;
     width: 100%;
+    font-variant-numeric: tabular-nums;
```

Comment:
look at https://css-tricks.com/almanac/properties/f/font-variant-numeric/

I think you also want to add `font-feature-settings: tnum`